src/Appium.Net/Appium/AppiumDriver.cs

```diff
@@ -227,6 +227,10 @@ public void Rotate(Dictionary<string, int> opts)
         public void RemoveApp(string appId) =>
             Execute(AppiumDriverCommand.RemoveApp, AppiumCommandExecutionHelper.PrepareArgument("appId", appId));
 
+        public void TerminateApp(string appId, int? timeout = null) =>
+            Execute(AppiumDriverCommand.TerminateApp, 
+                AppiumCommandExecutionHelper.PrepareArguments(new string[] { "appId", "options" }, new object[] { appId, timeout }));
```

Comment:
`timeout` should be a dictionary key

Reply:
I'm probably missing something obvious here, but I'm not exactly sure what you mean... Could you elaborate?

Reply:
`options` argument is expected to be a dictionary, where `timeout` is just one of the items. Although, `timeout` is being passed as `flat` value, which is not going to work

Reply:
Ah, I see. Changed it to a dictionary.